lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js

```diff
@@ -30,7 +30,9 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
       category: 'Performance',
       name: 'external-anchors-use-rel-noopener',
       description: 'Site opens external anchors using rel="noopener".',
-      helpText: 'Anchors that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener" target="_blank">opening page\'s performance does not suffer.</a>',
+      helpText: 'Open new tabs using <code>rel="noopener"</code> to improve performance and ' +
+                'prevent security vulnerabilities. <a href="https://developers.google.com/web/tools/lighthouse/audits/noopener" ' +
+                'target="_blank" rel="noopener noreferrer">Learn more</a>.',
```

Comment:
But on second thought, I'd like to collect referrer stats for the /web GA account, so I think I will remove it